session/build.gradle

```diff
@@ -1,5 +1,5 @@
-ext {
-    shadowJarEnabled = true
+plugins {
+    id "io.micronaut.build.internal.convention-core-library"
```

Comment:
why is session a core library? I assume you added because `shadowJarEnabled=true` but does it need to be a core library?

Reply:
That is simply remodeling what existed before, `session` was treated differently like the other `core-library`. I am not familiar enough with that module to say if it should be done differently.